Utilities/WDLs/PRSQC.wdl

```diff
@@ -0,0 +1,268 @@
+version 1.0
+
+# Simple PRS QC wdl for the PROGRESS VA project
+workflow PRSQC {
+    input {
+        String sample_name
+        File prs_control
+        File prs_sample
+        File control_thresholds
+        File sample_thresholds
+        File alphashape
+        Boolean manually_passed_control = false
+    }
+
+    Int cpu = 1
+    Int mem_gb = 4
+    Int disk_size_gb = 32
+    String docker = "us.gcr.io/broad-dsde-methods/python-data-slim:1.0"
+
+    if(!manually_passed_control) {
+        call CheckThresholds as CheckControl {
+            input:
+                scores = prs_control,
+                thresholds = control_thresholds,
+                cpu = cpu,
+                mem_gb = mem_gb,
+                disk_size_gb = disk_size_gb,
+                docker = docker
+        }
+
+        call DetectPCANovelties as DetectPCANoveltiesControl {
+            input:
+                test_sample = prs_control,
+                alphashape = alphashape,
+                cpu = cpu,
+                mem_gb = mem_gb,
+                disk_size_gb = disk_size_gb
+        }
+    }
+
+    call CheckThresholds as CheckSample {
+        input:
+            scores = prs_sample,
+            thresholds = sample_thresholds,
+            cpu = cpu,
+            mem_gb = mem_gb,
+            disk_size_gb = disk_size_gb,
+            docker = docker
+    }
+
+    call DetectPCANovelties as DetectPCANoveltiesSample {
+        input:
+            test_sample = prs_sample,
+            alphashape = alphashape,
+            cpu = cpu,
+            mem_gb = mem_gb,
+            disk_size_gb = disk_size_gb
+    }
+
+    call FinalizeQCOutputs {
```

Comment:
Do we need to spin up a whole new vm for this?  can't this just be 
```
qc_passed = qc_passed_control && qc_passed_sample && pca_qc_passed_control && pca_qc_passed_sample
```

Reply:
We certainly can. I might have overcomplicated this. Done